client/verta/verta/endpoint/build/_build.py

```diff
@@ -97,6 +99,35 @@ def message(self) -> str:
     def is_complete(self) -> bool:
         return self.status in ("finished", "error")
 
+    @property
+    def nvidia_gpu_compatible_hardware(self):
+        """Returns the list of this build's compatible Nvidia GPU models. If no hardware
+        compatibility was specified or if this build does not use GPUs, returns None.
+
+        .. versionadded:: 0.24.1
+
+        Returns
+        -------
+        set or None
+
+        """
+
+        hardware_compatibility = self._json["creator_request"].get("hardware_compatibility")
+        if hardware_compatibility is None:
+            return None
+        nvidia_gpu = hardware_compatibility.get("nvidia_gpu")
+        if nvidia_gpu is None:
+            return None
+
+        allows_all_models = nvidia_gpu.get("all")
+        allowed_models = set()
+        for model, allowed in nvidia_gpu.items():
```

Comment:
Is it guaranteed that all models will be present in the `nvidia_gpu` dict, even when they're `False?`

[Like](https://vertaai.slack.com/archives/CJ2MM54Q3/p1691708171921869?thread_ts=1691704020.401389&cid=CJ2MM54Q3)

```json
  "creator_request": {
    "hardware_compatibility": {
      "nvidia_gpu": {
        "T4": true
      }
    },
```